CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Contributing guidelines
+
+Thank you for wanting to contribute to this project!
+
+## Contributing code
+
+There are a couple of things we are going to look out for in PRs and knowing them upfront is going to reduce the number of times we will be going back and forth, making things more efficient.
+
+1. We have CI checks in place that validate formatting and code style.
+   Make sure `dprint check` and `cargo clippy` both finish without any warnings or errors on every commit.
```

Comment:
:question: Should we reference `just` or does that make things harder?

Reply:
thanks, I'll use `just` commands instead!